lib/doctest.js

```diff
@@ -559,3 +561,5 @@ function log(results) {
     }
   });
 }
+
+module.exports.log = log;
```

Comment:
I'm exporting these three functions so that the esm version can make use of them.